Creation of projects using the groupId/artifactId naming convention is incorrect

If I use chestnut to create a new project with a name like "tld.group/artifact", it doesn't do the right thing.

```
$ lein new chestnut com.lemondronor/airharp
Generating fresh Chestnut project.
README.md contains instructions to get you started.
$ lein test
Exception in thread "main" java.io.FileNotFoundException: Could not locate airharp/example_test__init.class or airharp/example_test.clj on classpath: , compiling:(/private/var/folders/mz/chqj8b3s2lsdb49hjxt37czw0000gn/T/form-init6089827244152768116.clj:1:124)
	at clojure.lang.Compiler.load(Compiler.java:7142)
	at clojure.lang.Compiler.loadFile(Compiler.java:7086)
	at clojure.main$load_script.invoke(main.clj:274)
	at clojure.main$init_opt.invoke(main.clj:279)
	at clojure.main$initialize.invoke(main.clj:307)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.doInvoke(main.clj:420)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
Caused by: java.io.FileNotFoundException: Could not locate airharp/example_test__init.class or airharp/example_test.clj on classpath: 
	at clojure.lang.RT.load(RT.java:443)
	at clojure.lang.RT.load(RT.java:411)
	at clojure.core$load$fn__5066.invoke(core.clj:5641)
	at clojure.core$load.doInvoke(core.clj:5640)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invoke(core.clj:5446)
	at clojure.core$load_lib$fn__5015.invoke(core.clj:5486)
	at clojure.core$load_lib.doInvoke(core.clj:5485)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invoke(core.clj:626)
	at clojure.core$load_libs.doInvoke(core.clj:5524)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invoke(core.clj:626)
	at clojure.core$require.doInvoke(core.clj:5607)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invoke(core.clj:626)
	at user$eval85.invoke(form-init6089827244152768116.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6703)
	at clojure.lang.Compiler.eval(Compiler.java:6693)
	at clojure.lang.Compiler.load(Compiler.java:7130)
	... 11 more
```

You can see that the namespace references are wrong:

```
$ grep -r 'com/lemon' .
./env/test/js/unit-test.js:    return com/lemondronor/airharp.test_runner.runner();
./test/clj/com/lemondronor/airharp/example_test.clj:(ns com/lemondronor/airharp.example-test
./test/cljs/com/lemondronor/airharp/core-test.cljs:(ns com/lemondronor/airharp.core-test
./test/cljs/com/lemondronor/airharp/test-runner.cljs:(ns com/lemondronor/airharp.test-runner
./test/cljs/com/lemondronor/airharp/test-runner.cljs:   [com/lemondronor/airharp.core-test]))
./test/cljs/com/lemondronor/airharp/test-runner.cljs:        'com/lemondronor/airharp.core-test))
```

PR #22 is related to this general topic.

--- a/src/leiningen/new/chestnut/test/clj/chestnut/example_test.clj
+++ b/src/leiningen/new/chestnut/test/clj/chestnut/example_test.clj
@@ -1,4 +1,4 @@
-(ns {{sanitized}}.example-test
+(ns {{project-ns}}.example-test
     (:require [clojure.test :refer :all]))
 
 (deftest example-passing-test

--- a/src/leiningen/new/chestnut/test/cljs/chestnut/core-test.cljs
+++ b/src/leiningen/new/chestnut/test/cljs/chestnut/core-test.cljs
@@ -1,4 +1,4 @@
-(ns {{sanitized}}.core-test
+(ns {{project-ns}}.core-test
   (:require-macros [cljs.test :refer (is deftest testing)])
   (:require [cljs.test]))
 

--- a/src/leiningen/new/chestnut/test/cljs/chestnut/test-runner.cljs
+++ b/src/leiningen/new/chestnut/test/cljs/chestnut/test-runner.cljs
@@ -1,4 +1,4 @@
-(ns {{sanitized}}.test-runner
+(ns {{project-ns}}.test-runner
   (:require
    [cljs.test :refer-macros [run-tests]]
    [{{sanitized}}.core-test]))

